index.js

```diff
@@ -64,6 +64,7 @@ function choo (opts) {
     const state = _store.state({state: {}})
 
     const tree = _router(state.location.href, state)
+    assert.ok(tree, 'tree should exist')
```

Comment:
could you change this to `choo.start: the router should always return a valid DOM node`; perhaps even do a `t.equal(typeof tree, 'object')` also :sparkles: